packages/@angular/cli/blueprints/resolver/files/__path__/__name__.resolver.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Resolve, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class <%= classifiedModuleName %>Resolver implements Resolve<any> {
+
+  constructor() { }
+
+  resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<any> {
```

Comment:
Maybe this should be like we did for guards and have the complete signature, i.e. `Observable<any> | Promise<any> | any` as return type